src/GradientValue.js

```diff
@@ -20,6 +24,17 @@
                                this.stops);
     }
 
+    getOutput() {
+      this.x0 = 0;
+      this.y0 = 0;
+      this.x1 = 256;
+      this.y1 = 0;
+      var gradient = this.toCanvasStyle();
+      this.ctx.fillStyle = gradient;
+      this.ctx.fillRect(0, 0, 256, 1);
+      return this.canvas;
```

Comment:
I guess this wrecks export all over again, but I'm not sure how we want to combat that.